3d_font.txt

```diff
@@ -433,3 +433,31 @@ z
         | \________/  /
         \ |        | /
          \|________|/
+4
+ ____       ____
+|\   \     |\   \
+| \   \    | \   \
+\  \   \___\__\   \
+ \  \              \
+  \  \___________   \
+   \ |          |\   \
+    \|__________| \   \
+                \  \   \
+                 \  \   \
+                  \  \___\
+                   \ |   |
+                    \|___|
+5
+   ___________________
```

Comment:
same here. 20 characters! looks too big. Maintain 14 chars, I don't see why this will give a problem.